### PR TITLE
glossary: Fix PDB tooltip text

### PR DESCRIPTION
**Pull request by disposedtrolley**
### Description

The tooltip text for the Pod Disruption Budget term renders incorrectly due to the templating of the "Pod" term within the `short_description`. This can be observed by hovering the cursor over the "PodDisruptionBudget" term on the following page: https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/

Screenshot 2024-10-28 at 15 37 55

Fix the issue by replacing the templated text with the literal word "Pod" for the English and Chinese versions of the documentation. Also slightly modify the wording to align with the full description on the following page: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets

### Issue

Closes: #

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
I recommend splitting it into 2 PRs so that each of the two teams can conduct a review.

**Comment by T-Lakshmi** [comment]
@disposedtrolley,
Thanks for your efforts in creating this PR. 
It appears that this PR includes changes to files across multiple languages English and Chinese. However, docs follows different processes for each localization, and we typically don't accept pull requests that impact multiple languages simultaneously. I would recommend modify only the primary website content (which means only English content) initially with this PR. After the merge of the English content, we could create separate pull request for the remaining language(s).

**Comment by T-Lakshmi** [comment]
Explicit hold until this PR only covers one language
(Ok to unhold once that's done)
/hold

**Comment by disposedtrolley** [comment]
@T-Lakshmi thanks for the review! I've modified the commit so it only changes the English documentation.

**Review by T-Lakshmi — APPROVED** [review]
/lgtm

Thanks for the PR @disposedtrolley

**Pull request merged**
